lib/bson/decimal128.rb

```diff
@@ -54,6 +54,11 @@ class Decimal128
     # @since 4.2.0
     NATIVE_TYPE = BigDecimal.freeze
 
+    # The error message if neither a String nor a BigDecimal are passed to #new.
+    #
+    # @since 4.2.0
```

Comment:
Yes, this is the public API and you can't make a private constant in Ruby. Thanks for pointing this out though, it's kind of an awkward design and inconsistent with the rest of the bson gem. I'll figure out how to do this differently.